lambdas/search/index.py

```diff
@@ -197,6 +201,7 @@ def lambda_handler(request):
         body=body,
         _source=_source,
         size=size,
+        from=es_from,
```

Comment:
it looks like `_from` and not `from` is the right kwarg? see docs: https://elasticsearch-py.readthedocs.io/en/6.3.1/api.html#elasticsearch.Elasticsearch.search

Also we should default to `0` not `None`?

Reply:
it's `from_`, and yes, we should default to `0` probably